data/json/items/comestibles.json

```diff
@@ -11622,6 +11622,35 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "flatbread",
+        "name" : "flatbread",
+        "weight" : 56,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 280,
+        "stim" : 0,
+        "container" : "bag_plastic",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 0,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 15,
```

Comment:
3 flours for 8*15=120 nutrition. How does it compare to similar food products, such as johnnycake?

That healthy 1 doesn't fit, fried flour with water isn't exactly healthy.